pod/perldelta.pod

```diff
@@ -143,36 +547,111 @@ XXX Remove this section if F<Porting/corelist-perldelta.pl> did not add any cont
 
 =head1 Documentation
 
-XXX Changes to files in F<pod/> go here. Consider grouping entries by
-file and be sure to link to the appropriate page, e.g. L<perlfunc>.
+=head2 Changes to Existing Documentation
 
-=head2 New Documentation
+We have attempted to update the documentation to reflect the changes
+listed in this document. If you find any we have missed, open an issue
+at L<https://github.com/Perl/perl5/issues>.
 
-XXX Changes which create B<new> files in F<pod/> go here.
+Additionally, the following selected changes have been made:
 
-=head3 L<XXX>
+=head3 L<perlapi>
 
-XXX Description of the purpose of the new file here
+=over 4
 
-=head2 Changes to Existing Documentation
+=item *
 
-We have attempted to update the documentation to reflect the changes
-listed in this document. If you find any we have missed, open an issue
-at L<https://github.com/Perl/perl5/issues>.
+Combined the documentation for several groups of related functions
+into single entries.
+
+=item *
 
-XXX Changes which significantly change existing files in F<pod/> go here.
-However, any changes to F<pod/perldiag.pod> should go in the L</Diagnostics>
-section.
+All forms of C<gv_fetchmeth()> are now documented together.
 
-Additionally, the following selected changes have been made:
+=item *
 
-=head3 L<XXX>
+C<gv_autoload4> is now documented with C<gv_autoload_pv> and additional notes added.
+
+=back
+
+=head3 L<perldata>
 
 =over 4
 
 =item *
 
-XXX Description of the change here
+Binary and octal floating-point constants (such as C<012.345p-2> and
+C<0b101.11p-1>) are now documented. This feature was first introduced in perl
+5.22.0 together with hexadecimal floating-point constants and had a few bug
+fixes in perl 5.28.0, but it was never formally documented.
+[L<GH #18664|https://github.com/Perl/perl5/issues/18664>]
+
+=head3 L<perlfunc>
+
+=over 4
+
+=item *
+
+Clarified the description of C<ref> and C<reftype> in relation
+to built-in types and class names.
+
+=item *
+
+Clarified that perl C<sort> is stable (and has been since v5.8.0).
+
+=back
+
+=head3 L<perlgov>
+
+=over 4
+
+=item *
+
+The list of Steering Council and Core Team members have been updated, following the conclusion of
+the latest election on 2024-07-17.
+
+=back
+
+=head3 L<perlguts>
+
+=over 4
+
+=item *
+
+Added some description of "real" C<AV>s compared to "fake" C<AV>s.
+
+=item *
+
+Documentation was updated to reflect that mixing C<Newx>, C<Renew>, and
+C<Safefree> vs C<malloc>, C<realloc>, and C<free> are not allowed, and mixing
+pointers between the 2 classes of APIs is not allowed. Updates made in
+L<perlguts> and L<perlclib>.
+
+=item *
+
+Additional caveats have been added to the description of C<TARG>.
+
+=back
+
+=head3 L<perlop>
+
+=over 4
+
+=item *
+
+Normalized alignment of verbatim sections, fixing how they are displayed by
+some Pod viewers that strip indentation.
```

Comment:
This doesn't make sense to me.  A viewer that strips indentation doesn't care about the original spacing.

I'd like to know of any viewers that don't strip the indentation and substitute their own.

Reply:
The content was using tabs for alignment of column data. But it was using 4 spaces for indents. If the indentation you strip is not aligned with a tab stop, then it changes the alignment of the later tabs.

You can see the result of this in the precedence table on https://perldoc.perl.org/perlop#Operator-Precedence-and-Associativity